docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
 
-In this step, you'll finally begin integrating dbt with Dagster! To do this, you'll:
+In this section, you'll finally begin integrating dbt with Dagster! To do this, you'll:
 
-- [Load the dbt models into Dagster as assets](#step-1-load-the-dbt-models-as-assets)
-- [Define a Dagster code location](#step-2-define-a-dagster-code-location)
-- [View the assets in the Dagster UI](#step-3-view-the-assets-in-the-dagster-ui)
+- [Install Dagster and the Dagster-dbt library](#step-0-install-dagster-and-the-dagster-dbt-library)
+- [Create a Dagster project that wraps your dbt project](#step-1-create-a-dagster-project-that-wraps-your-dbt-project)
+- [Inspect your Dagster project in Dagster's UI](#step-2-inspect-your-dagster-project-in-dagsters-ui)
+- [Build your dbt project, aka materialize your assets](#step-3-build-your-dbt-project-aka-materialize-your-assets)
+- [Review the Python code in your Dagster project](#step-4-review-the-python-code-in-your-dagster-project)
 
 ---
 
-## Step 1: Load the dbt models as assets
+## Step 0: Install Dagster and the Dagster-dbt library
 
-In this step, you'll load the dbt models into Dagster as assets using the [`dagster-dbt`](/\_apidocs/libraries/dagster-dbt) library.
+```shell
+pip install dagster dagster-webserver dagster-dbt
+```
 
-Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster/assets`, and add the following code:
+---
 
-```python
-from dagster_dbt import load_assets_from_dbt_project
+## Step 1: Create a Dagster project that wraps your dbt project
 
-from dagster import file_relative_path
+You can create a Dagster project that wraps your dbt project by using the `dagster-dbt` command line interface. Make sure you're in the directory where your dbt_project.yaml is. If you're continuing from the previous section, then you'll already be in this directory. Then, run:
 
+```shell
+dagster-dbt project scaffold --project-name jaffle_dagster
+```
 
-DBT_PROJECT_PATH = file_relative_path(__file__, "../../jaffle_shop")
+This creates a directory called `jaffle_dagster/` inside the current directory. The `jaffle_dagster/` directory contains a set of files that define a Dagster project.
 
-dbt_assets = load_assets_from_dbt_project(project_dir=DBT_PROJECT_PATH, key_prefix=["jaffle_shop"])
-```
+In general, it's up to you where to put your Dagster project. It's most common to put your Dagster project at the root of your git repository. So in this case, because the dbt_project.yml was at the root of the jaffle_shop git repository, we created our Dagster project there. `dagster_dbt project scaffold` creates the Dagster project in whatever directory you run it from. If that's a different directory from where your dbt_project.yaml lives, then you'll need to provide a value for the `--dbt-project-dir` option so that Dagster knows where to look for your dbt project.
```

Comment:
```suggestion
In general, it's up to you where to put your Dagster project. It's most common to put your Dagster project at the root of your git repository. In this case, because the `dbt_project.yml` was at the root of the `jaffle_shop` git repository, the Dagster project was created there. 

**Note**: The `dagster_dbt project scaffold` command creates the Dagster project in whatever directory you run it from. If that's a different directory from where your `dbt_project.yaml` lives, then you'll need to provide a value for the `--dbt-project-dir` option so that Dagster knows where to look for your dbt project.
```

Adding formatting for the file names, and suggesting we break this up to be less visually dense.

Also, what would you think about adding `**Note**:` to the beginning of the second paragraph? I think it brings more attention to what this is saying, versus it being lumped in with the first part of the explanation.

Reply:
> the Dagster project was created there. 

This is actually a choice that the user gets to make, rather than a decision the software automatically makes for them, so I think first person makes more sense. Let me know if you disagree.

Making the other edits recommended here.